.github/CONTRIBUTING.md

```diff
@@ -1,11 +1,30 @@
 # Prerequisites
 
-- [rust] toolchain
+- [rust] toolchain or [nix]
 
 [rust]: https://www.rust-lang.org
+[nix]: https://nixos.org/
 
 # Development
 
+Clone this repo into some place, e.g., `~/.vim-plugins`
+
+```sh
+mkdir -p ~/.vim-plugins
+cd ~/.vim-plugins
+git clone https://github.com/autozimu/LanguageClient-neovim.git
+cd LanguageClient-neovim
+git checkout dev
+# nix users can nix-shell at this point
```

Comment:
I think you missed a `use` here? `nix users can use nix-shell...` ?

Reply:
I meant it in the sense of can (execute) nix-shell, let me make it explicit